### PR TITLE
Added element print command to CLI

### PR DESCRIPTION
**Pull request by ori-moisis**
New CLI command allows printing specific values / elements from the workspace / state

---

Still missing tests for the command, will be added soon

---
_Release Notes_: 
CLI:
- Added new element print command

---
_User Notifications_: 
_None_

### CODE REVIEW

**Review by adamro — APPROVED** [review]

**Comment by ori-moisis** [comment]
This is ready for review now

File: packages/cli/src/commands/element.ts [hunk 1]
```diff
@@ -682,6 +682,82 @@ const renameElementsDef = createWorkspaceCommand({
...
+    selectors: validSelectors,
+    source: elementSource,
+    referenceSourcesIndex: await workspace.getReferenceSourcesIndex(),
+  })
+
+  await awu(relevantIds).forEach(async id => {
+    const value = await elementSource.get(id)
+    // We build a new static files source each time to avoid having to keep
+    // all the static files loaded in memory
+    const functions = nacl.getFunctions(staticFiles.buildInMemStaticFilesSource())
```
Comment (adamro) [inline packages/cli/src/commands/element.ts:716]: If the source is `state` isn't this an issue? (if there's a mis-match)

**Review by adamro — APPROVED** [review]
Yay!

File: packages/cli/src/commands/element.ts (hunk 1 above)
Comment (ori-moisis) [reply packages/cli/src/commands/element.ts:716]: this is the other direction
the "static files source" here is empty and is used so the "dump element" would put a file in it, we don't actually care about that file (which is why we build a new source each time as the comment says)

**Pull request merged**